catppuccin-steam.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash 
+
+# Install dependencies
+if [ -f /usr/bin/pacman ] 
+then
+  sudo pacman -S curl wget unzip git
+elif [ -f /usr/bin/emerge ]
+then
+  sudo emerge -av curl wget unzip git
+elif [ -f /usr/bin/apt ]
+then
+   sudo apt install curl wget unzip git
+elif [ -d /nix ]
+then
+  sudo nix-shell -p curl wget unzip git
```

Comment:
nix-shell doesnt need root to run, and nix has a [specific way](https://nixos.org/manual/nix/unstable/command-ref/nix-shell.html#use-as-a--interpreter) of handling scripts.

Reply:
sorry, the most i use nix is the fact that i have nixpkgs installed on arch